### ISSUE TITLE
Display outer variable in flat view

### ISSUE THREAD

**Issue by amzeratul** [Viladoman/StructLayout#10]
When viewing in cacheline mode, I find it useful to switch to flat view, so it fits the screen.

However, it currently shows the innermost objects, which can lead to some structures being difficult to understand. For example, in the screenshot below, it'll show the internal members of std::vector, which to me is less useful than the name of the vector itself.

![image](https://user-images.githubusercontent.com/4175191/97512050-6f5ff500-1980-11eb-9fdc-555985daf4eb.png)

I'd like to request an alternate flat view, that shows the outermost variable, or something else to that effect.

**Comment by Viladoman** [comment]
I think you have a good point!. 

The Top level flat gives better high level information to the user and the bottom gives what actually is stored in memory.
I just did a quick mock up:  

![image](https://user-images.githubusercontent.com/8118743/97528004-15146380-1983-11eb-8cd8-c43ab4c81a10.png)

Basically I made the buttons 'Expand All' & 'Collapse All' available also in flat mode so we can choose between Top and Bottom Flat within the same mode. 

I will clean this up a little bit and bundle it in the next Release (v0.2.3). 

Thanks for suggestion.
